doc/getting-started/apl-nuc.rst

```diff
@@ -364,6 +364,89 @@ For hugeTLB to work, you'll need to reserve huge pages:
      by using a 2M page, but make sure your huge page reservation size is
      large enough for your usage.
 
+
+USB Device Sharing
+==========================================
+
+The ACRN hypervisor supports USB device sharing.  Suppose you have
+two keyboards and mice connected to your device, one keyboard and 
+mouse set for the SOS, and the other set for the UOS.  
+
+1. Boot the SOS and plug in the two keyboards and two mice
```

Comment:
As discussed in previous comments, you'll need to plug in one device and run ``dmesg`` to get the device info needed.  Then plug in the next device and run ``dmesg`` again.   Have you verified that, after doing this, and updating launch_uos.sh, if you reboot the NUC with the four devices attached that the device info is the same? (I assume if you remove one or more of the devices the device info changes.)

Reply:
@dbkinder  user should get USB device info one by one, but here i want to make documentation clear and don't want to have 4 "dmesg" log ourput and list them one by one, showing 4 USB device info one time as  the example here only. 
 
reboot NUC with the four devices attached  that the device info is the same.